scripts/variantstore/wdl/ImportGenomes.wdl

```diff
@@ -525,9 +527,55 @@ task LoadTable {
     fi
 
     if [ $NUM_FILES -gt 0 ]; then
-        bq load --location=US --project_id=~{project_id} --skip_leading_rows=1 --source_format=CSV -F "\t" $TABLE $DIR$FILES ~{schema} || exit 1
-        echo "ingested ${FILES} file from $DIR into table $TABLE"
-        gsutil -m mv $DIR$FILES ${DIR}done/
+        # get list of of pet files and their byte sizes
+        echo "Getting load file sizes(bytes) and path to each file."
+        gsutil du "${DIR}${FILES}" | tr " " "\t" | tr -s "\t" > ~{datatype}_du.txt
+
+        # get total memory in bytes
+        echo "Calculating total files' size(bytes)."
+        TOTAL_FILE_SIZE=$(awk '{print $1}' OFS="\t" ~{datatype}_du.txt| paste -sd+ - | bc)
+
+        # get number of iterations to loop through file - round up to get full set of files
+        num_sets=$(((TOTAL_FILE_SIZE+16492674416639)/16492674416640))
```

Comment:
Why is this 16000000000000 below but something else here?

Reply:
good catch. i wanted to se the TB limit to something a bit lower than the max allowed just to be on the safe side and didn't swap it here.